src/main/java/io/stargate/sgv2/jsonapi/api/model/command/serializer/ColumnDescSerializer.java

```diff
@@ -35,7 +35,8 @@ public void serialize(
           jsonGenerator.writeStringField(
               TableDescConstants.ColumnDesc.VALUE_TYPE, st.valueType().getApiName());
       case VectorColumnDesc vt -> {
-        jsonGenerator.writeNumberField(TableDescConstants.ColumnDesc.DIMENSION, vt.getDimensions());
+        jsonGenerator.writeNumberField(
+            TableDescConstants.ColumnDesc.DIMENSION, vt.getDimensions().orElse(null));
```

Comment:
Since we use `orElse(null)`, would it be easier to use plain `Integer` instead of `Optional<Integer>`?